### PR TITLE
Fix a bug that LR is not correctly scaled when initial LR is not 1.0

### PR DESCRIPTION
**Pull request by noobpwnftw**
Simplify unnecessary variable.

### CODE REVIEW

File: src/learn/learn.cpp [hunk 1]
```diff
@@ -986,15 +984,17 @@ namespace Learner
 
         if (sr.sfen_for_mse.size() && done)
         {
+            cout << "INFO: "
```
Comment (nodchip) [inline src/learn/learn.cpp:987]: Do we need to change this section?

Comment (noobpwnftw) [reply src/learn/learn.cpp]: No, I will remove it.

**Review by nodchip — APPROVED** [review]
Thank you for fixing.

**Comment by nodchip** [comment]
Thank you for the pull request.  I merged it.

**Comment by nodchip** [comment]
The CI test failed in the master.  But it failed in bench command.  It would not be related to this pull request.

**Comment by noobpwnftw** [comment]
Might due to failed resource download, thanks for checking.

**Pull request merged**